src/dohye1/pages/Store/StoreDetail.tsx

```diff
@@ -1,20 +1,41 @@
 ﻿import { useParams } from 'react-router-dom';
-import MenuList from '../../component/MenuList';
-import { getMenuGroupList } from '../../utils';
+import MenuListComp from '../../component/MenuList';
 import { TEST_ID } from '../../constant';
+import { useEffect, useState } from 'react';
+import { useStoreOperationTime } from '../../hooks/useMenu';
 
 export default function StoreDetail() {
   const { storeId } = useParams();
 
-  const menuList = getMenuGroupList(['인기메뉴', '포케', '보울']);
+  const [isOpen, setIsOpen] = useState(false);
+
   const title = storeId;
 
+  const { data, isFetching } = useStoreOperationTime(storeId);
+
+  useEffect(() => {
+    if (data) {
+      const { openHour, closeHour } = data;
+
+      const curHour = new Date(Date.now()).getHours();
+
+      if (curHour >= openHour && curHour < closeHour) {
+        setIsOpen(true);
+      }
+    }
+  }, [data]);
```

Comment:
useEffect 안써도 될 것 처럼 보이네요!

```suggestion
if (!data) {
  return null;
  }
  
  const curHour = new Date(Date.now()).getHours();
  const isOpen = curHour >= openHour && curHour < closeHour;
```